vmdb/spec/models/ems_refresh/refreshers/foreman_refresher_spec.rb

```diff
@@ -0,0 +1,131 @@
+require "spec_helper"
+
+describe EmsRefresh::Refreshers::ForemanRefresher do
+  # where clause to use spec related entries
+  let(:spec_related) { "name like 'ProviderRefreshSpec%'" }
+  let(:provider) do
+    _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
+    FactoryGirl.create(:provider_foreman,
+                       :zone       => zone,
+                       :url        => "10.8.96.102",
```

Comment:
We have a helper method for creating "random" IPs (see other factories or the factory_girl_helper).  However, if the test requires a specific IP for VCR, that should just be in the test like we do with all of the other refresher specs (and then anoynmized)